test/integration/admin_login_test.rb

```diff
@@ -177,7 +177,7 @@ def setup
     click_link_or_button('Orders')
 
     within("#ordered") do
-      select 'completed', from: 'update_order_status[status]'
+      select 'completed', from: "update_order_status[status_#{@order.id}]"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.